istio/assets/monitors/failed_sidecar_injection.json

```
@@ -0,0 +1,29 @@
+{
+	"name": "[Istio] Increased number of failed sidecar injection requests",
+	"type": "metric alert",
+	"query": "sum(last_5m):avg:istio.sidecar_injection.requests_total{*}.as_count() - avg:istio.sidecar_injection.success_total{*}.as_count() > 3",
+	"message": "Monitor Istio for failed sidecar injection requests. Increased failed requests may signify other issues in your Istio mesh and requires attention.",
```

```suggestion
	"message": "Monitor Istio for failed sidecar injection requests. Increasing failed requests may signify other issues in your Istio mesh and requires attention.",
```
Less sure about this change, either way seems to work grammatically.